floodviz/templates/index.html

```diff
@@ -10,40 +11,45 @@
 	<link rel="stylesheet" type="text/css" href="{{ url_for('static', filename='css/map.css') }}" />
 	<link rel="stylesheet" type="text/css" href="{{ url_for('static', filename='css/peakflow.css') }}" />
 	<link rel="stylesheet" type="text/css" href="{{ url_for('static', filename='css/hydrograph.css') }}" />
-	<title>Flood Viz</title>
-	<script type="application/ld+json">
-		{{ linked_data | tojson | safe }}
-
-	</script>
+	<script type="application/ld+json"> {{ linked_data | tojson | safe }} </script>
+	<script> var FV = FV || {}; </script>
+	{% if config.GA_ID != "null" %}
+		<script> FV.GA_RUN = true; </script>
+	{% else %}
+		<script> FV.GA_RUN = false; </script>
+	{% endif %}
```

Comment:
Let's work on this together once you get in. You shouldn't have to create the variable FV.GA_RUN. Rather use the if block to conditionally include the google analytics script.